### PR TITLE
Add Grafana container with Prometheus integration and basic dashboard

### PR DESCRIPTION
**Pull request by platini2**

### CODE REVIEW

**Comment by herlesupreeth** [comment]
Hey @platini2 thank you for the MR. I am on vacation currently will get back to this once back

File: 4g-external-ims-deploy.yaml [hunk 1]
```diff
@@ -319,6 +319,23 @@ services:
...
+    build: ./grafana
+    image: docker_grafana
+    container_name: grafana
+    env_file:
+      - .env
+    volumes:
+      - ./grafana:/mnt/grafana
+      - /etc/timezone:/etc/timezone:ro
+      - /etc/localtime:/etc/localtime:ro
+    expose:
```
Comment (herlesupreeth) [inline 4g-external-ims-deploy.yaml:332]: Is grafana using port 3000 by default?

Comment (platini2) [reply 4g-external-ims-deploy.yaml:332]: Yes Grafana default port is 3000

**Comment by herlesupreeth** [comment]
Thank you once again for your contribution

**Pull request merged**